cypress/integration/bodySpec.js

```diff
@@ -50,7 +50,10 @@ describe('Article Body Tests', () => {
   });
 
   it('should render a title', () => {
-    cy.title().should('eq', "Meghan's bouquet laid on tomb of unknown warrior");
+    cy.title().should(
```

Comment:
Do we want to test the persian article for it's title?

Reply:
Hi @jtart, yes. @jamesbrumpton will be updating this.